docs/core/transactions/fees.md

```diff
@@ -1,130 +0,0 @@
----
```

Comment:
While I agree that fees are confusing and scattered throughout the docs, is there a single place you'd put them? I would avoid deleting this page until the _one page to rule them all_ is created

Reply:
added back under Advanced Concepts